config/production.yaml

```diff
@@ -57,6 +57,10 @@ NYPL_LOCATIONS_BY_CODE: https://nypl-core-objects-mapping-qa.s3.amazonaws.com/by
 # API_CLIENT_ID and API_CLIENT_SECRET must be configured in secrets file
 NYPL_API_CLIENT_TOKEN_URL: https://isso.nypl.org/oauth/token
 
+# DRB API Credentials
+DRB_API_HOST: '0.0.0.0'
```

Comment:
I believe this should be `digital-research-books-beta.nypl.org` Remember that these are URLs that clients will use to connect to our servers so they need to be a resolvable domain.

Reply:
This makes sense. Thanks for the correction.